src/commands/general/lookup.ts

```diff
@@ -0,0 +1,44 @@
+import Command, { SendFunction } from '../../util/BaseCommand';
+import SentinelClient from '../../client/SentinelClient';
+import { Message } from 'discord.js';
+import CommandArguments from '../../util/CommandArguments';
+import CommandError from '../../structures/CommandError';
+import { URLs } from '../../util/Constants';
+import fetch from 'node-fetch';
+
+export default class LookupCommand extends Command {
+	constructor(client: SentinelClient) {
+		super(client, {
+			aliases: ['geo'],
+			name: 'lookup',
+			dmAllowed: true,
+			description: 'Lookup a IP'
+		}, __filename);
+	}
+
+	async run(message: Message, args: CommandArguments, send: SendFunction) {
+		const ip = args[0];
+		const valid = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(ip);
```

Comment:
this regex should be moved to the constants file, so is there a shorter version of this? see if perhaps you can find one